src/utils/dom.ts

```diff
@@ -135,22 +135,24 @@ interface GetSlottedOptions {
   selector?: string;
 }
 
+const defaultSlotSelector = "> :not([slot])";
+
 export function getSlotted<T extends Element = Element>(
   element: Element,
   slotName: string,
   options: GetSlottedOptions & { all: true }
 ): T[];
 export function getSlotted<T extends Element = Element>(
   element: Element,
-  slotName: string,
+  slotName?: string,
   options?: GetSlottedOptions
 ): T | null;
 export function getSlotted<T extends Element = Element>(
   element: Element,
-  slotName: string,
+  slotName?: string,
```

Comment:
Can you update the signature to support both

```ts
getSlotted(el, slotName, options);
getSlotted(el, options);
```

?

Reply:
Yeah, or we can just update the slotName to be apart of options? What do you think?

Do you have an example of making the second param be identified as the third?